deploy/infrastructure/modules/ecr_public/main.tf

```diff
@@ -0,0 +1,10 @@
+provider "aws" {
+  alias  = "us_east_1"
```

Comment:
Use [existing](https://github.com/filecoin-project/storetheindex/blob/main/deploy/infrastructure/common/main.tf#L3) alias instead of re-declaring AWS provider.